examples/project-foundry/foundry/script/Deploy.s.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Script, console2 as console} from "forge-std/Script.sol";
+import {Counter} from "../src/Counter.sol";
+
+contract CounterScript is Script {
+    function setUp() public {}
+
+    function run() public {
+        vm.broadcast();
+        Counter counter = new Counter();
+
+        console.log("Deployments:");
+        console.log("Counter deployed at: ", address(counter));
```

Comment:
Could we also add the ponder reload cURL command here, like the docs suggest?

Reply:
I added it to the README. Let me know if that works better for you.